src/React/Resources/assets/src/render_controller.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import React, {ReactElement} from 'react';
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    readonly componentValue: string;
+    readonly propsValue: object;
+
+    static values = {
+        component: String,
+        props: Object,
+    };
+
+    connect() {
+        this._dispatchEvent('react:connect', { component: this.componentValue, props: this.propsValue });
+
+        // Use a timeout to avoid mounting and demounting right after
```

Comment:
why would it mount and then demount right after without the timeout?

Reply:
https://stackoverflow.com/questions/66618415/why-is-my-stimulus-js-controller-firing-twice

Turbo with cache will trigger two connects in a row (cache then real) which means rendering the whole component again in a short time if the page renders quickly. I used this technique to circumvent the issue, maybe we don't want to care :) .

Reply:
Maybe should we at least provide a way to disable this (or maybe could we add this later if needed)? I fear that it may be the cause of some edge cases.